src/main/java/com/rasklaad/wiremock/metrics/MetricsConfiguration.java

```diff
@@ -67,12 +68,20 @@ public MetricsConfiguration maximumMetricExpectedValue(Double maximumMetricExpec
         return this;
     }
 
+    public MetricsConfiguration minimumMetricExpectedValue(Double minimumMetricExpectedValue) {
+        this.minimumMetricExpectedValue = minimumMetricExpectedValue;
+        return this;
+    }
+
     static MetricsConfiguration defaultConfiguration() {
         return new MetricsConfiguration()
             .totalTimeMetricEnabled(true)
             .processingTimeMetricEnabled(true)
             .serveTimeMetricEnabled(true)
             .responseSendTimeEnabled(true)
+            .registerNotMatchedRequests()
+            .minimumMetricExpectedValue(1d)
+            .maximumMetricExpectedValue(5000d)
```

Comment:
I don't like idea of changing default configuration. It can lead to unexpected behavior for users, when they update library.